tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+import pandas as pd
+import pytest
+
+@pytest.fixture
+def stock_prices_df() -> pd.DataFrame:
+    return pd.read_csv('tests/stock_prices.csv', parse_dates=True, index_col='date')
```

Comment:
I pulled the stock prices data from https://github.com/robertmartin8/PyPortfolioOpt/blob/master/tests/stock_prices.csv. Let me know if you think there is a better dataset.

We'll probably need some sample data for our tests.

Reply:
Also, for those not familiar with `conftest.py` files, this just allows us to share common fixtures across all of our tests.